.github/workflows/release.yaml

```diff
@@ -29,9 +29,9 @@ jobs:
         with:
           go-version: 1.13
       - run: make setup
-      - run: make build TOPOLVM_VERSION=${{ steps.check_version.outputs.version }}
+      - run: make build/lvmd TOPOLVM_VERSION=${{ steps.check_version.outputs.version }}
       - run: tar czf lvmd-${{ steps.check_version.outputs.version }}.tar.gz -C ./build lvmd
-      - run: make image
+      - run: make image TOPOLVM_VERSION=${{ steps.check_version.outputs.version }}
```

Comment:
Specified TOPOLVM_VERSION is not used in the `image` target in Makefile.
To use the variable, you also have to modify Makefile and dockerfile.